packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -56,11 +66,28 @@
                     : undefined,
         },
     ]
-
-    let modal: Modal
-    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
     $: $selectedAccountVestingPayouts, (timeUntilNextPayout = getTimeUntilNextPayout())
 
+    onMount(() => {
+        getMinStorage()
+        areOutputsReadyForConsolidation()
+    })
+
+    async function areOutputsReadyForConsolidation(): Promise<boolean> {
```

Comment:
I think that this function, instead of returning a boolean, could change the value of a variable in a `then`
Something like that
```suggestion
    let hasOutputsToConsolidate: boolean =  false
    async function areOutputsReadyForConsolidation(): Promise<void> {
        $selectedAccount.prepareConsolidateOutputs({ force: false, outputThreshold: 2 })
            .then(() => hasOutputsToConsolidate = true)
            .catch(() => hasOutputsToConsolidate = false)
    }
```